QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -0,0 +1,49 @@
+package org.team1540.quasarhelios;
+
+import ccre.channel.BooleanInputPoll;
+import ccre.channel.BooleanOutput;
+import ccre.channel.EventInput;
+import ccre.channel.FloatInputPoll;
+import ccre.channel.FloatOutput;
+import ccre.channel.FloatStatus;
+import ccre.ctrl.BooleanMixing;
+import ccre.ctrl.EventMixing;
+import ccre.ctrl.FloatMixing;
+import ccre.ctrl.PIDControl;
+import ccre.igneous.Igneous;
+
+public class Clamp {
+	public final BooleanOutput openControl;
+	public final FloatOutput heightControl;
+	
+	private FloatStatus min;
+	private FloatStatus max;
+	
+	public Clamp() {
+		
+		BooleanOutput leftClamp = Igneous.makeSolenoid(0);
+		BooleanOutput rightClamp = Igneous.makeSolenoid(1);
```

Comment:
Consider not defining these separately if you don't need to use them separately.